signer/src/api/info.rs

```diff
@@ -655,10 +652,11 @@ mod tests {
                 settings.signer.signer_round_max_duration = Duration::from_secs(2);
                 settings.signer.bitcoin_presign_request_max_duration = Duration::from_secs(3);
                 settings.signer.dkg_max_duration = Duration::from_secs(4);
-                settings.signer.sbtc_bitcoin_start_height = Some(101);
+                settings.signer.sbtc_bitcoin_start_height = Some(BitcoinBlockHeight::from(101u64));
```

Comment:
It seems we mostly use `.into()` for the conversions, is there a reason to prefer the explicit version here (and below for `dkg_min_bitcoin_block_height`)?

Reply:
I used explicit `BitcoinBlockHeight::from` instead of `into()` in cases where rust is not smart enough to understand into which type it should cast. However, we even bumped rust version since I did most of my changes, so maybe in some cases it can be rewrited to `.into()`

Reply:
Resolving since even if the `into` would work, the explicit one does seem better being this a test